__tests__/subscription.test.ts

```diff
@@ -1,48 +1,64 @@
-import TestSubscription from './pubsub/subscriptions/example.subscription-ts';
+import { Subscribers } from '../src/subscriber';
+import SubscriptionService from '../src/service/subscription';
+
+const mockPubSub = jest.fn();
 
-const mockTopic = jest.fn().mockImplementation(() => ({
-  get: jest.fn(),
-  publish: jest.fn(),
-}));
-const mockPublish = jest.fn();
-const mockSubscribe = jest.fn();
 jest.mock('@google-cloud/pubsub', () => ({
   __esModule: true,
-  PubSub: jest.fn().mockImplementation(() => ({
-    topic: mockTopic,
-    publish: jest.fn(),
-  })),
+  PubSub: mockPubSub,
 }));
 
-jest.mock('../src/service/pubsub', () => ({
+const mockPublish = jest.fn();
+jest.mock('../src/service/pubsub', (): any => ({
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8106467)
```suggestion
jest.mock('../src/service/pubsub', (): unknown => ({
```